src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -45,4 +45,9 @@ cd $__TOOLS_DIR/tool-runtime/
 $__DOTNET_CMD restore --source https://www.myget.org/F/dotnet-core/ --source https://www.myget.org/F/dotnet-buildtools/ --source https://www.nuget.org/api/v2/
 $__DOTNET_CMD publish -f dnxcore50 -r ${__PUBLISH_RID} -o $__TOOLRUNTIME_DIR
 
+if [ -n $BUILDTOOLS_OVERRIDE_RUNTIME ]; then
+    find $__TOOLRUNTIME_DIR -name *.ni.* | xargs rm 2>/dev/null
+    cp -R $BUILDTOOLS_OVERRIDE_RUNTIME/* $__TOOLRUNTIME_DIR
```

Comment:
I believe you also need to copy the runtime into the $__TOOLRUNTIME_DIR/dotnetcli/runtime/coreclr/ folder in order for dotnet cli to work